realm-jni/src/io_realm_internal_TableQuery.cpp

```diff
@@ -859,21 +861,270 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_TableQuery_nativeFind(
     return -1;
 }
 
+JNIEXPORT jlong JNICALL Java_io_realm_internal_TableQuery_nativeFindWithHandover(
+    JNIEnv* env, jobject, jlong bgSharedGroupPtr, jlong replicationPtr, jlong queryPtr, jlong fromTableRow) {
+
+    SharedGroup::Handover <Query> *handoverQueryPtr = reinterpret_cast<SharedGroup::Handover <Query> * > (queryPtr);
```

Comment:
Casing in general doesn't seem to be aligned. My current understanding is that variables coming from Java should be "camelCase", everything else should use "_". I'll skip further comments on this, but see #1489, and we should probably have an discussion on this also vs. the style guide from Core.